src/vmm/src/builder.rs

```diff
@@ -65,84 +65,54 @@ use crate::vstate::vm::Vm;
 use crate::{device_manager, EventManager, RestoreVcpusError, Vmm, VmmError};
 
 /// Errors associated with starting the instance.
-#[derive(Debug, thiserror::Error)]
+#[derive(Debug, thiserror::Error, displaydoc::Display)]
 pub enum StartMicrovmError {
-    /// Unable to attach block device to Vmm.
-    #[error("Unable to attach block device to Vmm: {0}")]
+    /// Unable to attach block device to Vmm: {0}
     AttachBlockDevice(io::Error),
-    /// This error is thrown by the minimal boot loader implementation.
-    #[error("System configuration error: {0:?}")]
+    /// System configuration error: {0}
     ConfigureSystem(crate::arch::ConfigurationError),
-    /// Error using CPU template to configure vCPUs
-    #[error("Failed to create guest config: {0:?}")]
+    /// Failed to create guest config: {0}
     CreateGuestConfig(#[from] GuestConfigError),
-    /// Internal errors are due to resource exhaustion.
-    #[error("Cannot create network device. {}", format!("{:?}", .0).replace('\"', ""))]
+    /// Cannot create network device. {0}
```

Comment:
can replace full stop with a colon for uniformity

Reply:
fixed